### DISCUSSION TITLE
controlling window size when using multiple windows

### DISCUSSION THREAD

**Discussion by jkissi** [marcomusy/vedo#603] [Q&A] [answered]
Hi marco!! 
Hope you had a good holiday and the beginning of your year has been good. I had to focus on a different area of my research for a bit but now I'm back with more questions about vedo!! 
I have a couple of buttons on my plot but I would like to move them to a seperate (but attached) window. I know the shape keyword has the "/" and "|" characters but I was wondering: is there any way to conrtol the size of the windows so they are different? So if I use "2/1" as the argument, can I make the first horizontal window 80% width and the secpnd 20%?

Cheers
j

**Comment by marcomusy** [comment, marked as answer]
Hi @jkissi  welcome back :)

you have many ways to control for that, one simple way is in settings:
`vedo.setttings.windowSplittingPosition=0.80`

but you can also rearrange any nr of renderers anywhere in the plotter window, check out:

https://github.com/marcomusy/vedo/blob/master/examples/basic/multirenderers.py
https://github.com/marcomusy/vedo/blob/master/examples/advanced/multi_viewer2.py

**Reply by jkissi** [reply]
Hi Marco
At first I wasn't really sure what you were showing me, then I had a look at the code and thought about for a bit and realised that this is amazing!! I didn't realise that such fine grain control was possible. I'm pretty sure this will allow me to do what I want but if I haveany more questions, I'll be back. 

Thanks, as always, 
j
